src/sagemaker/amazon/amazon_estimator.py

```diff
@@ -400,6 +400,7 @@ def registry(region_name, algorithm=None):
             "us-east-1": "785573368785",
             "eu-west-1": "802834080501",
             "us-east-2": "007439368137",
+            "ap-northeast-1": "941853720454",
```

Comment:
Can we use `sagemaker.model. NEO_IMAGE_ACCOUNT` here?
https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/model.py#L46